### PR TITLE
added codeowners

### PR DESCRIPTION
**Pull request by wjcunningham7**
- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation, VERSION, and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by wjcunningham7** [comment]
Documentation on CODEOWNERS [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

tl;dr specify who's required to approve a PR that modifies certain files.

**Review by Emmanuel289 — DISMISSED** [review]
Changelog is updated based on added codeowners and version number has been bumped from .12 ~> .13. Looks good to me.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.24.13] - 2022-02-11
+
+### Added
+
+- CODEOWNERS for certain files
```
[the remark this answers is not in the document]
Comment (wjcunningham7) [reply CHANGELOG.md]: @Emmanuel289 made the change!

[the remark this answers is not in the document]
Comment (Emmanuel289) [reply CHANGELOG.md]: @wjcunningham7 Thanks!

**Pull request merged**